ui/src/app/pages/data/data-set/dataset-page.tsx

```diff
@@ -719,15 +736,24 @@ const DataSetPage = fp.flow(withCurrentWorkspace(), withUrlParams())(
                   fontSize: '20px', fontWeight: 400}}>
                   Select cohorts, concept sets, and values above to generate a preview table
                 </div>
-                <Button data-test-id='preview-button' disabled={this.disableSave()}
-                        style={{marginTop: '0.5rem', height: '1.8rem', width: '6.5rem'}}
-                        onClick={() => this.getPreviewList()}>
-                  Preview Table
-                </Button>
               </div>
             }
           </div>
         </FadeBox>
+        <div>
+          <div style={styles.footer} />
```

Comment:
I haven't patched this locally but please test this very carefully locally. Sticky footers are very difficult to get right in my experience.

Please try alternate window sizes when testing.